app.js

```diff
@@ -560,6 +561,25 @@ var eventsModule = function () {
             //   values (CSV), into a hierarchy using d3.stratify.
             var root = EFsToHierarchy();
 
+            var efDurationSum = function(efNode) {
+                if (efNode.children === undefined) {
+                    // Node has no children, return duration value
+                    return efNode.data.durationMinutes;
+                } else {
+                    // Node has children, compute duration of each child
+                    var childSum = 0;
+                    for (var i = 0; i < efNode.children.length; i++) {
+                        childSum += efDurationSum(efNode.children[i]);
+                    }
+                    return childSum;
+                }
+            }
+
+            var totalTime = efDurationSum(root);
+
+            var $totalTimeElement = $('.exele-total-time', symbolElement);
+            $totalTimeElement[0].innerHTML = 'Total event time: ' + totalTime.toFixed(2);
```

Comment:
For someone newer to JS can you elaborate on "easier"?  My thinking is that jQuery is eventually just going to hit the `innerHTML` anyway and it is the same number of lines of code so how do I choose the preferred syntax?  We can discuss AFK.